R/fotmob_players.R

```diff
@@ -114,9 +115,13 @@
     "away_team_color" = .ppc(p, "teamData", "away", "color", n = n)
   )
 
-  rows$stats <- vector(mode = "list", length = nrow(stats))
-  for (i in 1:nrow(rows)) {
-    rows[i, ]$stats <- list(stats[1, ])
```

Comment:
i had accidentally left this hard-coded as `stats[1, ]` when it should be `stats[i, ]`. this resolves the issue noted [here](https://github.com/JaseZiv/worldfootballR/issues/298#issuecomment-1644913918)